keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/EnvironmentArtifactPin.kt

```diff
@@ -18,3 +18,18 @@ data class PinnedEnvironment(
   val artifact: DeliveryArtifact,
   val version: String
 )
+
+data class EnvironmentArtifactVeto(
+  val deliveryConfigName: String,
+  val targetEnvironment: String,
+  val reference: String,
+  val type: String,
+  val version: String
+)
+
+data class EnvironmentArtifactVetoes(
```

Comment:
Any reason why we can't just use `List<EnvironmentArtifactVeto>` instead of this class?